.nvmrc

```diff
@@ -1 +1 @@
-16.14.0
```

Comment:
why was this made?

Reply:
to fix the warnings, i think if you run the npm i you will see the problem with 16.14.0